x-pack/plugins/security_solution/cypress/test_plans/detection_response/prebuilt_rules/prebuilt_rules_install_update_workflows.mdx

```diff
@@ -0,0 +1,190 @@
+- Most of our users are on the 7.17.x version, that’s why the 8.x version is specified on scenarios, because this TestPlan is considering a minimum version of 8.x.
+- The rule Customization feature should be available to users on the Basic license and higher.
+
+
+### Scenarios
+
+### Notifications
+
+#### **Scenario: No callout messages are displayed when user does not have prebuilt rules installed**
+
+**GIVEN** user doesn't have any 8.x prebuilt rules installed  
+**AND** user is running a fresh instance  
+**WHEN** user navigates to the Rules Management Page  
+**THEN** no callouts message should be displayed
+
+#### **Scenario: No callout messages are displayed when there are no pending installs/updates**
+
+**GIVEN** user has the latest version of prebuilt rules `<prebuilt_rules_status>`  
+**WHEN** user navigates to the Rules Management Page  
+**THEN** no callout message is displayed for `<prebuilt_rules_status>` rules  
+
+*CASE 1: `<prebuilt_rules_status>` = installed*  
+*CASE 2: `<prebuilt_rules_status>` = updated*
+
+#### **Scenario: Callout message is displayed when there are new prebuilt rules available to install**
+
+**GIVEN** user already has 8.x prebuilt rules installed  
+**AND** there are new prebuilt rules available to install  
+**WHEN** user navigates to the Rules Management Page  
+	**THEN** user should see a callout message to install new prebuilt rules  
+**AND** the number of new rules available to install should be displayed on the +Add Elastic Rules link  
+
+#### **Scenario: Callout message is displayed when there are new updates on already installed prebuilt rules**
+
+**GIVEN** user already has 8.x prebuilt rules installed  
+**AND** there are new updates available for those prebuilt rules  
+**WHEN** users navigate to the Rules Management Page  
+**THEN** users should see an update callout message  
+**AND** the number of outdated rules should be displayed on the Rules Updates tab  
+
+#### **Scenario: User is notified of available prebuilt rules to install when a rule is deleted**
+
+**GIVEN** user has the latest version of prebuilt rules installed  
+**WHEN** user navigates to Rules Management Page  
+**AND** user deletes some prebuilt rules  
+**THEN** user should see a callout message with the same amount of prebuilt rules ready to install  
+
+
+
+
+### Prebuilt Rules Installation
+
+#### **Scenario: User without any installed prebuilt rule can install `<amount>` prebuilt rules**
+
+**GIVEN** a user that doesn’t have prebuilt rules installed  
+**WHEN** user navigates to Add Elastic Rules Page  
+**THEN** available prebuilt rules are displayed on Elastic Rules table  
+**AND** user can install `<amount>` prebuilt Rules  
+**AND** successfully installed message is displayed after installation  
+**AND** installed rules are removed from Elastic Rules table  
+**AND** rules to install counter is decreased accordingly  
+
+*CASE 1: `<amount>` = All*  
+*CASE 2: `<amount>` = Selected*  
```

Comment:
Let's add:
```
*CASE 3: `<amount>` = Individual*
```
To also cover the installation of a rule by clicking on the "Install rule" button within the row.

Reply:
Synced with @vgomez-el: we'll add this case.